docs/docs/contributing-guide/setup/ubuntu.md

```diff
@@ -24,6 +24,11 @@ Follow these steps to setup and run ToolJet on Ubuntu. Open terminal and run the
     sudo apt install postgresql postgresql-contrib
     sudo apt-get install libpq-dev
     ```
+    
+    1.3 Install PostgREST
```

Comment:
Can put as (optional)

And below we can say - Required only if TooljetDb is used.

Reply:
I have added the changes.